src/js/pages/ServicesPage.js

```diff
@@ -53,9 +53,13 @@ var ServicesPage = React.createClass({
   },
 
   render: function () {
+    // Make sure to grow when logs are displayed
+    let routes = this.context.router.getCurrentRoutes();
+
     return (
       <Page
         navigation={this.getNavigation()}
+        dontScroll={routes[routes.length - 1].dontScroll}
```

Comment:
I would really like to create a task to get that done outside of this PR. This PR has already dragged along for some time now and it doesn't feel like a part of this task. I created a task for it